src/moneywiz_api/model/transaction.py

```diff
@@ -225,15 +271,15 @@ def __init__(self, row):
         super().__init__(row)
         self.account = row["ZACCOUNT2"]
         self.amount = RDH.get_decimal(row, "ZAMOUNT1")
-        self.reconcile_amount = RDH.get_decimal(row, "ZRECONCILEAMOUNT")
+        self.reconcile_amount = RDH.get_nullable_decimal(row, "ZRECONCILEAMOUNT")
```

Comment:
any evidence that the reconcile_amount is nullable?

Reply:
It is funny, but I have only one transaction like this, dated Nov/2023, in ETH. I use reconciliation with amount 0 and the corresponding comment to indicate that, at this point, the account balance is zeroed
```
  {'ZACCOUNT2': 5748,
   'ZAMOUNT1': -2.401342176793775e-12,
   'ZCURRENCYEXCHANGERATE': 1.0,
   'ZDATE1': 722458351.69891,
   'ZDESC2': 'New\xa0balance\xa0=\xa00',
   'ZFEE2': 0.0,
   'ZFLAGS1': 0,
   'ZGID': '3F45DED0-6AB2-4CA4-A77F-CBFE34985684-1826-0000005AF14672E0',
   'ZINVESTMENTHOLDING': 4068,
   'ZNOTES1': '',
   'ZNUMBEROFSHARES1': -1.1657341758564144e-15,
   'ZOBJECTCREATIONDATE': 722458351.703616,
   'ZORIGINALAMOUNT': -1.1657341758564144e-15,
   'ZORIGINALCURRENCY': 'ETH',
   'ZORIGINALEXCHANGERATE': 0.0,
   'ZORIGINALFEE': 0.0,
   'ZPRICEPERSHARE1': 0.0,
   'ZRECONCILED': 1,
   'ZRECONCILENUMBEROFSHARES': 0.0,
   'ZSTATUS1': 2,
   'ZSYMBOL1': 'ETH',
   'ZVOIDCHEQUE': 0,
   'Z_ENT': 42,
   'Z_OPT': 2,
   'Z_PK': 24495}
```